### PR TITLE
Correct cmake flags passing

### LINKED ISSUE #18214: Cannot open include file: 'numa.h': No such file or directory
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

I want to build libtorch for windows x64 by using build_windows.bat. I disabled numa and glog but still got the following failure. Could you give some hints how to fix it?

D:\Code\GitHub\pytorch\CMakeFiles\CMakeTmp\src.cxx(1): fatal error C1083: Cannot open include file: 'numa.h': No such file or directory [D:\Code\GitHub\pytorch\CMakeFiles\CMakeTmp\cmTC_3a810.vcxproj]

D:\Code\GitHub\pytorch\CMakeFiles\CMakeTmp\src.cxx(1): fatal error C1083: Cannot open include file: 'glog/stl_logging.h': No such file or directory [D:\Code\GitHub\pytorch\CMakeFiles\CMakeTmp\cmTC_d2d67.vcxproj]

### PR DESCRIPTION
**Pull request by peterjc123**
Fixes #18214.

According to the CMake manual, we should pass the arguments first, and put the directory as the last element. Otherwise, these flags may not be passed correctly.

Reference: 
1. https://cmake.org/cmake/help/latest/manual/cmake.1.html#synopsis
2. https://stackoverflow.com/a/27169347

### CODE REVIEW

File: scripts/build_windows.bat [hunk 1]
```diff
@@ -61,7 +61,7 @@ cd %CAFFE2_ROOT%\build
 
 :: Set up cmake. We will skip building the test files right now.
 :: TODO: enable cuda support.
-cmake .. ^
+cmake ^
```
Comment (orionr) [inline scripts/build_windows.bat]: So long as we don't use build_windows.bat in any of our CI testing I'd recommend removing it.

**Review by orionr — APPROVED** [review]
If tests pass happy to bring this in. cc @kostmo as well.

**Review by ezyang — APPROVED** [review]

File: scripts/build_windows.bat (hunk 1 above)
Comment (peterjc123) [reply scripts/build_windows.bat]: It seems that it is used in the CI `pr/caffe2-py2-cuda9.0-cudnn7-windows-build`.

**Pull request closed without merging**